### DISCUSSION TITLE
How to use @grafana/ui in standalone react app?

### DISCUSSION THREAD

**Discussion by LordZardeck** [grafana/grafana#66082] [General]
The UI library seems to have some really cool components, and I thought I'd try them out in my react app. But I can't seem to get it to work, despite following the storybook code. What setup do I need to do in order to use these components in a react app?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
